### PR TITLE
Remove singleton usage in scripts

### PR DESCRIPTION
**Pull request by thc202**
Replace singletons with the injected variables (`model` and `control`).

---
Ruby scripts were not yet changed, pending update of the library.

### CODE REVIEW

**Review by kingthorin — APPROVED** [review]

File: addOns/graaljs/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Added
 - Add info URL.
 
+### Changed
+- Replace usage of singletons with injected variables (e.g. `model`, `conrol`) in scripts.
```
Comment (ricekot) [inline addOns/graaljs/CHANGELOG.md:11]: There's a typo in `control`.

File: addOns/groovy/CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Changed
 - Update minimum ZAP version to 2.12.0.
+- Replace usage of singletons with injected variables (e.g. `model`, `conrol`) in scripts.
```
Comment (ricekot) [inline addOns/groovy/CHANGELOG.md:10]: This one too.

Comment (kingthorin) [reply addOns/groovy/CHANGELOG.md:10]: Nice catch!

File: addOns/graaljs/CHANGELOG.md (hunk 1 above)
Comment (thc202) [reply addOns/graaljs/CHANGELOG.md:11]: I thought I had fixed them all…

Comment (thc202) [reply addOns/graaljs/CHANGELOG.md]: Done.

**Review by psiinon — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Comment by ricekot** [comment]
Thanks!

**Pull request merged**